data_structures/linked_list/__init__.py

```diff
@@ -1,19 +1,28 @@
+"""
+Linked Lists consists of Nodes.
+Nodes contain data and also may link to other nodes:
+    - Head Node: First node, the address of the
+                 head node gives us access of the complete list
+    - Last node: points to null
+"""
+
+
 class Node:
-    def __init__(self, item, next):
+    def __init__(self, item: any, next):
```

Comment:
There is no type hint called `any`. You probably meant `Any` which is imported from the `typing` module.

Also, please provide the type hint for the `next` parameter which probably will also be `Any` and return type hint as well which will be `None`.